tests/foreman/cli/test_oscap.py

```diff
@@ -104,8 +104,10 @@ def setUpClass(cls):
             remote_file="/tmp/{0}".format(cls.file_name)
         )
         cls.title = 'rhel-6-content'
-        result = Scapcontent.info({'title': cls.title})
-        if not result['title'] in cls.title:
+        result = Scapcontent.list()
+        result = [scap['title'] for scap in result if
+                  scap.get('title') in cls.title]
+        if result is not cls.title:
```

Comment:
I cannot understand this line, we are checking if a result (that is a list) is not in a title (that is a string)?
(I think python should raise exception here)
I was assuming
```python
if not result:
    # create a an new one
```

Reply:
Will fix this, it should be 
```python
if cls.title not in result:
    # create a an new one
```